frontend/fingertips-frontend/app/pill/page.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Pill } from '@/components/molecules/Pill';
+
+/**
+ * This is a dummy page that is being used to test the component
+ * Should be removed once the component is needed elsewhere
+ */
+
+export default function Page() {
+  return (
+    <div>
+      <Pill selectedFilterName="Greater Manchester ICB - OOT sub-location" />
+      <Pill selectedFilterName="Dementia" />
+    </div>
+  );
+}
```

Comment:
I'm happy for this page to be removed now. It was good enough to see how it shows up when the component is used.